R/diagnostics.R

```diff
@@ -240,15 +265,18 @@ plot.psis_loo <- function(x,
             "% of Pareto k estimates are Inf/NA/NaN and not plotted.")
   }
 
-  if (diagnostic == "n_eff") {
+  if (diagnostic == "ESS" | diagnostic == "n_eff") {
```

Comment:
```suggestion
  if (diagnostic == "ESS" || diagnostic == "n_eff") {
```

Doesn't really matter here because `match.arg` is called above and that returns a single value, but `||` is used when comparing single values (and enforces that requirement) and `|` is used when comparing vectors